dir.props

```diff
@@ -33,6 +33,7 @@
     <!-- Output directories -->
     <BinDir Condition="'$(BinDir)'==''">$(ProjectDir)bin/</BinDir>
     <ObjDir Condition="'$(ObjDir)'==''">$(BinDir)obj/</ObjDir>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'==''">$(ObjDir)</BaseIntermediateOutputPath>
```

Comment:
Minor but I would prefer to keep this with the other IntermediateOutputPath properties where it was before. I understand it is used in BuildVersion.targets but is there any reason BuildVersion.targets needs to be imported in this location and not at the bottom of the dir.props file?